index.html

```diff
@@ -529,28 +529,50 @@ <h2>
         Examples
       </h2>
       <p>
-        This example shows the current screen orientation to the console every
-        time the screen orientation state changes.
+        In this example for a mobile phone, clicking the landscape button makes
+        a request to go fullscreen and then lock to landscape orientation. Once
+        it is in landscape, the unlock button will take the screen orientation
+        back to default (portrait) or the portrait button will also change it
+        back to portrait.
+      </p>
+      <p>
+        The log shows the change in orientation type and angle, and whether the
+        event type is load or change.
       </p>
       <pre class='example html'>
 &lt;script&gt;
-  var show = function() {
-     console.log("Orientation type is " + screen.orientation.type);
-     console.log("Orientation angle is " + screen.orientation.angle);
-  }
+async function lockLandscape(){
+  await document.documentElement.requestFullscreen() 
```

Comment:
```suggestion
  await document.documentElement.requestFullscreen();
```

Reply:
I think we might actually want to do the `checkIfFullScreen()` function, otherwise, it looks like you always have to call requestFullScreen() before using the API. 

For `checkIfFullScreen()`, you will need to figure out how to check if fullscreen is enabled (MDN should probably be able to tell you). Basically, if fullscreen is already enabled, then just return, otherwise, call requestFullscreen(); and return the resulting promise.